camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/SpecParameterUnwrappingTest.java

```diff
@@ -158,18 +157,18 @@ public void testDepentantCatalogsInheritParameters(Class<? extends BrooklynObjec
                 "      - simple",
                 "  - id: " + SYMBOLIC_NAME,
                 "    item:",
-                "      type: paramItem:" + TEST_VERSION);
+                "      type: paramItem");
```

Comment:
Don't remember; I think it just didn't work properly with it set, and other tests didn't have it?